eng/scripts/appium-install.ps1

```diff
@@ -0,0 +1,23 @@
+node -v
+npm install -g appium@next
```

Comment:
I think we need to include a call here to update npm as well

When trying to run appium from VS I was getting the following error

```
2023-05-05 14:10:50:602 [Appium] npm version must be at least >=8; current is 6.14.12. Run "npm install -g npm" to upgrade.
```


After running `npm install -g npm` everything is working for me now

Reply:
Ok  I added that.